src/Utility/LogStream.C

```diff
@@ -62,8 +62,10 @@ void utl::LogStream::addExtraLog(std::shared_ptr<std::ostream> extra, bool clear
 int utl::LogStream::precision(int streamsize)
 {
   int result = streamsize;
-  if (m_out)
-    result = m_out->precision(streamsize);
+  if (m_out) {
+    result = m_out->precision();
```

Comment:
yes, was fooled by that the diff view above did not show the whole change. So all good then